__tests__/endpoints.test.js

```diff
@@ -103,4 +103,37 @@ describe('Test Endpoints', () => {
               });
         });
     });
+    describe('/api/articles/:article_id/comments', () => {
+        test('200 - responds with comments array related to the particular article', () => {
+            return request(app)
+            .get("/api/articles/3/comments")
+            .expect(200)
+            .then(({ body }) => {
+                expect(body).toBeInstanceOf(Array);
```

Comment:
Would be better if check the length of the comments. Also when send in the controller, we'd like to send an object with a descriptive key, so in the test we've got a property to refer to. Also when the users get the data back from the api, they know exactly what they've got. 
`expect(body.comments).....`